packages/firebase/firestore.rules

```diff
@@ -193,5 +193,10 @@ service cloud.firestore {
       }
       // #endregion attendance-checks
     }
+    // #region public-info-check
+      match /publicOrgInfo/{organization} {
+      allow read: if true;
+      }
+    // #endregion public-info-check
```

Comment:
Please watch the indentation here (prettier doesn't work in `.rules` files), otherwise this will get really messy really quickly